cnxauthoring/tests/test_functional.py

```diff
@@ -3300,14 +3311,14 @@ def test_publish_derived_from_binder(self):
 
         publication_info = response.json
         publication_id = publication_info['publication']
-        self.assertEqual(publication_info['state'], 'Done/Success')
+        self.assertEqual(publication_info['state'], 'Waiting for moderation')
```

Comment:
Two questions:

1. Are these the only places where 'Waiting for moderation' will be expected?  Or is it possible to get the waiting message for every response from publishing.  Because from my local machine its coming up in several places. Or have @karenc other changes fixed this?

2.  Is it still possible to receive 'Done/Success' from publishing?  Is this the publication_info['state'] always guaranteed to be 'Waiting for moderation'